fix(styles): removes unwanted margin for heading in layouts/portal (#2407)
swisspost/design-system: .changeset/spotty-countries-smoke.md

---
'@swisspost/design-system-styles': patch
---

> Removes unwanted margin from heading when using `@use "@swisspost/design-system-styles/index.scss";`

Reviewer: ```suggestion
Removes unwanted margin from heading within the notification overlay component.
```
